tools/smartos-release

```diff
@@ -75,34 +76,48 @@ fi
 
 set -o errexit
 
-# Note that ${BRANCH} appears only once here as we assume that for release
-# builds, all project branches and smartos-live itself use the same branch
-# name. See $(PUB_BRANCH_DESC) in the top-level Makefile for more details.
-SOURCE=/${MANTA_USER}/public/builds/platform/${BRANCH}-${TIMESTAMP}/platform/
 SMARTOS=/${MANTA_USER}/public/SmartOS
-DESTINATION=${SMARTOS}/${TIMESTAMP}
+SMARTOS_RELEASE=${SMARTOS}/${TIMESTAMP}
+BITS=output/bits/platform
 
 start_time=$(date +%s)
-echo "Creating release snaplinks under ${DESTINATION}"
 
-mmkdir -v -p ${DESTINATION}
-mfind ${SOURCE} -t o | while read OBJECT; do
-    mln ${OBJECT} ${DESTINATION}/$(basename ${OBJECT})
+echo "Uploading release objects to ${SMARTOS_RELEASE}"
+mmkdir -p ${SMARTOS_RELEASE}
+
+mput -f $BITS/platform-${BRANCH}-${TIMESTAMP}.tgz ${SMARTOS_RELEASE}/platform-${BRANCH}-${TIMESTAMP}.tgz
+mput -f $BITS/platform-${BRANCH}-${TIMESTAMP}.imgmanifest ${SMARTOS_RELEASE}/platform-${BRANCH}-${TIMESTAMP}.imgmanifest
+mput -f $BITS/boot-${BRANCH}-${TIMESTAMP}.tgz ${SMARTOS_RELEASE}/boot-${BRANCH}-${TIMESTAMP}.tgz
+mput -f $BITS/tests-${BRANCH}-${TIMESTAMP}.tgz ${SMARTOS_RELEASE}/tests-${BRANCH}-${TIMESTAMP}.tgz
+
+mput -f $BITS/smartos-${TIMESTAMP}.iso ${SMARTOS_RELEASE}/smartos-${TIMESTAMP}.iso
+mput -f $BITS/smartos-${TIMESTAMP}-USB.img.gz ${SMARTOS_RELEASE}/smartos-${TIMESTAMP}-USB.img.gz
+mput -f $BITS/smartos-${TIMESTAMP}.vmwarevm.tar.gz ${SMARTOS_RELEASE}/smartos-${TIMESTAMP}.vmwarevm.tar.gz
+
+ADDITIONAL_FILES="SINGLE_USER_ROOT_PASSWORD.txt
```

Comment:
Can we please do the same - in the same loop - for :88-95 ? They're all the same file name now.

Reply:
Oh absolutely - thanks!